src/store/auth/operations/loginOperation.js

```diff
@@ -0,0 +1,40 @@
+import { createLogic } from 'redux-logic';
+import { LOGIN_SUBMIT, LOGIN_LOADING, LOGIN_ERROR, LOGIN_SUCCESS } from '../actions/types';
+import { API_KEY } from '../../../constants';
+
+export const loginOperation = createLogic({
+    type: LOGIN_SUBMIT,
+    latest: true,
+    async process({ action, axios }, dispatch, done) {
+        dispatch({type: LOGIN_LOADING, loading: true});
+        dispatch({type: LOGIN_ERROR, error: false, errorTxt: null});
+        let sessionID;
+        const {username, password} = action;
+        try {
+            const requestToken = await axios.get(`/authentication/token/new?api_key=${API_KEY}`)
```

Comment:
can you can add `api_key` to every request using axios instance?

Reply:
added